### PR TITLE
Add installation instructions and component requirements to the user guide

### PR DESCRIPTION
**Pull request by KtorZ**
Screenshot from 2020-04-22 17-08-35

---

Screenshot from 2020-04-22 17-08-41

---

Screenshot from 2020-04-22 17-08-48

---

Screenshot from 2020-04-22 17-09-00

### CODE REVIEW

File: user-guide/content/docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+| Repository                                                           | Tags                          | Documentation               |
+| :---                                                                 | :---:                         | :---:                       |
+| [inputoutput/cardano-node][inputoutput-cardano-node]                 | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-node]    |
+| [inputoutput/cardano-db-sync][inputoutput-cardano-db-sync]           | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-db-sync] |
+| [inputoutput/cardano-graphql][inputoutput-cardano-graphql]           | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-graphql] |
+| [inputoutput/cardano-explorer-api][inputoutput-cardano-explorer-api] | `master`, `MAJOR.MINOR.PATCH` | [link][doc-cardano-rest]    |
```
Comment (piotr-iohk) [inline user-guide/content/docs/installation.md:15]: Documentation for cardano-node and cardano-db-sync link to `docker.nix`... I fear of those links a bit... (cause it is part of live, moving code)
`cardano-graphql` links to https://github.com/input-output-hk/cardano-graphql/wiki/Docker which does not exist atm, oh well.

File: user-guide/content/docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+weight: 3
+title: Installation Instructions
+---
+
+## Using Docker (recommended)
```
Comment (piotr-iohk) [inline user-guide/content/docs/installation.md:6]: Hm, if that's so, more effort needs to be put in testing this.

File: user-guide/content/docs/installation.md (hunk 1 above)
Comment (KtorZ) [reply user-guide/content/docs/installation.md:15]: Yeah, the graphql one still lives in a branch... And, there's no Docker documentation for cardano-node and cardano-db-sync apart from these bits in the nix file written by rhys... I've asked the teams already to write some but... * sounds of silence *

File: user-guide/content/docs/installation.md (hunk 2 above)
Comment (KtorZ) [reply user-guide/content/docs/installation.md:6]: What type of efforts do you imagine? Would you be able to capture these as stories and discuss it with Alex?

**Pull request merged**